src/app/modules/form/admin-form/admin-form.service.ts

```diff
@@ -1148,3 +1169,241 @@ const isMobileFieldUpdateAllowed = (
     },
   )
 }
+
+export const createTwilioCredentials = (
+  twilioCredentials: TwilioCredentials,
+  formId: string,
+): ResultAsync<unknown, ApplicationError> => {
+  const twilioCredentialsData: TwilioCredentialsData =
+    new TwilioCredentialsData(twilioCredentials)
+  const msgSrvcName = `formsg/${config.secretEnv}/form/${formId}/twilio`
+  const body: CreateSecretRequest = {
+    Name: msgSrvcName,
+    SecretString: twilioCredentialsData.toString(),
+  }
+
+  logger.info({
+    message: `no msgSrvcName, creating Twilio credentials for form ${formId}`,
+    meta: {
+      action: 'createTwilioCredentials',
+      formId: formId,
+      msgSrvcName,
+    },
```

Comment:
since this is reused, may want to extract it to a `logMeta` variable like what we do for many of our functions